src/main/encryptors/Encryptor.ts

```diff
@@ -21,18 +21,39 @@ export interface DecryptResult<T> {
   decryptedItem: T
 }
 
-export type EncryptedJayZItem<T, U extends keyof T> = ItemWithEncryptedFields<T, U> & {
-  __jayz__metadata: EncryptedItemMetadata<T, U>
+export type EncryptedJayZItem<T, U extends keyof T> = LegacyEncryptedJayZItem<T, U> | EncryptedJayZItemV1<T, U>
```

Comment:
Since we want to preserve the ability to decrypt old data, we need to differentiate between the old format and the new format.